src/services/addressResolver/index.test.ts

```diff
@@ -0,0 +1,20 @@
+import { getIdentityName } from "./index";
```

Comment:
Dont test against a live endpoint:
1. It's slow
2. Undeterministic test

Instead, mock the axios call and simulate different situations.